Misc/NEWS.d/next/Library/2020-05-15-17-38-21.bpo-40479.yamSCh.rst

```diff
@@ -0,0 +1 @@
+The :mod:`hashlib` now compiles with OpenSSL 3.0.0-alpha2.
```

Comment:
Technically, it's the _hashlib module, since hashlib is written in pure Python :-D

```suggestion
The :mod:`_hashlib` now compiles with OpenSSL 3.0.0-alpha2.
```

Reply:
That makes sense, it just more a remark than a request ;-)